### DISCUSSION TITLE
Integration of discussions-ui in a portal

### DISCUSSION THREAD

**Discussion by shoaib-mohmad** [Sunbird-Lern/Community#174] [Query]
Is there any documentation which will help to integrate [discussions-ui](https://github.com/Sunbird-Lern/discussions-UI) library into a portal, after setting up node bb & middleware.

**Comment by AmiableAnil** [comment]
@shoaib-mohmad Please follow this link https://lern.sunbird.org/use/developer-guide/discussion-forum
Let me know if it is helpful or not.
